src/GitVersionCore/FileLocking/Abstractions/IFileLock.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+
+namespace GitVersion.FileLocking.Abstractions
```

Comment:
We aren't using `Abstractions` anywhere else in the codebase and I don't think they provide any useful information. Please remove.

```suggestion
namespace GitVersion.FileLocking
```